tests/unit_tests/test_server_client.py

```diff
@@ -307,6 +307,42 @@ async def test_serve_playground_with_api_router() -> None:
     assert response.status_code == 200
 
 
+async def test_root_path_on_playground(event_loop: AbstractEventLoop) -> None:
+    """Test that the playground respects the root_path for requesting assets"""
+
+    for root_path in ("/home/root", "/home/root/"):
```

Comment:
What's the purpose of looping twice through the same root path?

Reply:
I introduced some logic around slash stripping/joining, so though it was worth ensuring it continues to work with and without the trailing slash.

Reply:
ah got it -- i didn't notice the extra `/`